lib/mix_unused/filter.ex

```diff
@@ -24,23 +24,35 @@ defmodule MixUnused.Filter do
   @spec reject_matching(exports :: Exports.t(), patterns :: [pattern()]) ::
           Exports.t()
   def reject_matching(exports, patterns) do
-    filters =
-      Enum.map(patterns, fn
-        {_m, _f, _a} = entry -> entry
-        {m, f} -> {m, f, :_}
-        {m} -> {m, :_, :_}
-        m when is_atom(m) -> {m, :_, :_}
-        %Regex{} = m -> {m, :_, :_}
-        cb when is_function(cb) -> cb
-      end)
-
-    Enum.reject(exports, fn {func, meta} ->
-      Enum.any?(filters, &mfa_match?(&1, func, meta))
+    Map.reject(exports, matcher(patterns))
```

Comment:
questa ed un paio di `Map.filter` qui sotto ed in un altro file andrebbero cambiati per mantenere la compatibilità con elixir ~1.10 (dichiarata dalla libreria), visto che sono stati aggiunti nella 1.13

```suggestion
   exports |> Enum.reject(matcher(patterns)) |> Enum.into(%{})
```